tooling/composer-cache

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env bash
+
+set -e
+
+COMPOSER_WARMUP=~/.dd-trace-php-composer-warmup
+
+function usage {
+    echo "Usage:"
+    echo "  ./composer-update-all.sh cache"
+    echo "  ./composer-update-all.sh link-cache"
+    echo "  ./composer-update-all.sh unlink-cache"
+    echo
```

Comment:
This says `composer-update-all`, but the file is named `composer-cache`; usually these match.